content/en/docs/components/pipelines/operator-guides/installation/_index.md

```diff
@@ -24,4 +24,11 @@ You should be familiar with [Kubernetes](https://kubernetes.io/docs/home/),
      kubectl apply -k "github.com/kubeflow/pipelines/manifests/kustomize/env/dev?ref=$PIPELINE_VERSION"
      ```
 
-     The Kubeflow Pipelines deployment requires approximately 3 minutes to complete.
\ No newline at end of file
+     The Kubeflow Pipelines deployment requires approximately 3 minutes to complete.
+
+2. Run the following to port-forward:
+     ```
+     kubectl port-forward -n kubeflow svc/ml-pipeline-ui 8080:80
+     ```
+
+3. Open [kubeflow-pipelines](http://localhost:8080)
```

Comment:
I think we can leave just the address here and leave the obvious explicit.

```suggestion
3. Open http://localhost:8080 on your browser to see the Kubeflow Pipelines UI.
```

Reply:
+1